docs/reference/contributing/guidelines/workflow.md

```diff
@@ -139,6 +141,20 @@ A breaking change is any change that results in breaking user space. It should h
 
 A contribution containing a breaking change is the most difficult PR to get merged. Any breaking changes in a codebase can have a large negative impact on any users of the codebase. Breaking changes are always limited to a major version release.
 
+Every pull request with breaking change must contain a release notes section called "Release notes" to describe the changes to users.
+
+It must contain:
+
+- A brief description of changes introduced, including justification description.
+- An analysis of effects: components affected, potential consequences for users and reasons for breaking user space.
```

Comment:
Isn't there some duplication here? We request "justification description" under the "brief description" section, and again "reasons for breaking user space" under "analysis of effects". The reasons for breaking user space are part of the justification, not really to do with how it effects users, IMO.

Reply:
I'll remove `and reasons for breaking user space.` from here

Reply:
Fixed in 252826d